_content/av/anti-muslim-movements-sri-lanka-and-myanmar_walton-matthew-j.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Anti-Muslim Movements in Sri Lanka and Myanmar: Connections and Commonalities"
+authors:
+  - "Matthew J Walton"
+external_url: "https://podcasts.ox.ac.uk/anti-muslim-movements-sri-lanka-and-myanmar-connections-and-commonalities"
+drive_links:
+  - "https://drive.google.com/file/d/1mnB1oeCyByAHzjMsZeBssj4L8hwqwReu/view?usp=sharing"
+course: theravada-roots
+tags:
+  - sri-lanka
+  - burma
+  - ideology 
```

Comment:
🤔 Let's be more specific than `ideology`  Could be any of:

- extremism
- religion
- nationalism
- race